dgraphHelper.js

```diff
@@ -0,0 +1,370 @@
+import diff from 'deep-diff';
+import dgraph from 'dgraph-js';
+import grpc from 'grpc';
+
+import { SCHEMA, TYPES } from './schema';
+
+const DB_URL = process.env.DGRAPH_URL;
+
+const create_client = () => {
+  const CLIENT_STUB = new dgraph.DgraphClientStub(DB_URL, grpc.credentials.createInsecure());
```

Comment:
Same as SCHEMA & TYPES, eslint error :(

Reply:
eslint complain that you are shadowin a variable, so if someone read your code it could be tricked or doesn't know easily form where the variable came from: the global scope or the argument
So you just need another name, or remove you global SCHEMA & TYPES :smile: (This option is better, global state is bad)